crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -444,7 +444,11 @@ fn container_top_level_properties(
                         );
                     });
 
-                container.set_kind(container_kind);
+                if container.kind() != container_kind {
+                    viewport
+                        .blueprint
+                        .set_container_kind(*tile_id, container_kind);
+                }
```

Comment:
My preference would be to do this no-op check inside of `set_container_kind` any way so no reason to do it twice.